### PR TITLE
Hotfix/media queries

### PR DESCRIPTION
**Pull request by catherine-jones**

### CODE REVIEW

**Comment by tobyhede** [comment]
So this refactors out some pixel values into variables?
Seems ok to me.

**Comment by tobyhede** [comment]
Dont think this counts as a hotfix, btw. A hotfix is something you intend to push to production immediately. 

Its a refactor, maybe?

**Comment by catherine-jones** [comment]
@tobyhede I do want to push it to prod straight away if that's ok?

File: client/src/dashboard/styles/_base.scss [hunk 1]
```diff
@@ -93,19 +93,19 @@ h4 {
 }
 
 .hide-on-desktop {
-  @media (min-width: 769px) {
+  @media (min-width: $min-width-screen-md) {
```
Comment (elisechant) [inline client/src/dashboard/styles/_base.scss]: we need to review this implementation since introducing Bootstrap's Grid system

**Comment by elisechant** [comment]
@catherine-jones why close?

**Pull request closed without merging**